docs/src/pages/components/tabs/tabs.md

```diff
@@ -91,6 +91,29 @@ Tab labels may be either all icons or all text.
 
 {{"demo": "pages/components/tabs/IconLabelTabs.js", "bg": true}}
 
+## Accessibility
+
+(WAI-ARIA: https://www.w3.org/TR/wai-aria-practices/#tabpanel)
+
+This section lists necessary steps to provide necessary information for assistive technology:
+
+1. label `Tabs` via `aria-label` or `aria-labelledby`
+2. `Tab`s need to be connected to their
+corresponding `[role="tabpanel"]` by setting the correct `id`, `aria-controls` and `aria-labelledby`
+
+An example for the current implementation can be found in the demos on this page. We've also published an experimental API in `@material-ui/lab` that does not require
+extra work.
+
+### Keyboard navigation
+
+The components implement keyboard navigation using the "manual activation" behavior. If you want to switch to the
+"selection automatically follows focus" behavior you have pass `selectionFollowsFocus` to the `Tabs` component. The WAI-ARIA authoring practices have a detailed guide on [how to decide then to make selection automatically follow focus](https://www.w3.org/TR/wai-aria-practices/#kbd_selection_follows_focus).
+
+### Demo
```

Comment:
What do you think of having the new demo focusing on `selectionFollowsFocus`? We can observe the manual activation with the other?

```suggestion
### selectionFollowsFocus
```

Reply:
The demo is part of the tabs accessibility story. It's not solely focused on automatic activation behavior.

Reply:
Right, my concern would be around what the new demo adds relatively to the others.

Reply:
Maybe: if we only had the Tabs (no panels) but with the two versions rendered (removing the need for a toggle).

Reply:
You're focusing too much on 

```diff
-<Tabs />
+<Tabs selectionFollowsFocus />
```

The more important aspect is trying out different activation behaviors. Demos are not just about the code.

Reply:
Agree, I think that it would be great to show the two tabs, side by side with both behaviors:

```jsx
<Tabs />
<Tabs selectionFollowsFocus />
```

👌

Reply:
Interesting point. But I'd say seeing the panel switch is somewhat important so that people are more aware of what kind of additional render work `selectionFollowsFocus` adds.

Reply:
Fair. My assumption was that we could display an `active tab` value bellow the two tabs.